doc/selectors.rst

```diff
@@ -142,4 +142,31 @@ It is possible to use user defined vectors as a basis for the selectors. For exa
     result = cq.Workplane("XY").box(10, 10, 10)
 
     # chamfer only one edge
-    result = result.edges(">(-1, 1, 0)").chamfer(1)
+    result = result.edges('>(-1, 1, 0)').chamfer(1)
```

Comment:
I'm surprised that black isn't complaining about the single quotes here.

Reply:
My understanding was PR #1220 was a manual black run (or blacken-docs).  It is not automated.